docs/src/man/getting_started.md

```diff
@@ -877,6 +877,58 @@ package provides interfaces similar to LINQ and [dplyr](https://dplyr.tidyverse.
 
 See the [Data manipulation frameworks](@ref) section for more information.
 
+#### Selecting rows with `filter` and `subset`
+
+We have seen above how to subset a `DataFrame` to several criteria, involving multiple columns,
+by supplying a logical vector to the first dimension.
+For instance, in the following we want to subset to all rows where `x > 2` and where `a == 'c'`:
+
+```jldoctest dataframe
+julia> df = DataFrame(x=1:4, y=["Alice", "Bob", "Claire", "Dylan"], a='a':'d')
+4×3 DataFrame
+ Row │ x      y       a    
+     │ Int64  String  Char 
+─────┼─────────────────────
+   1 │     1  Alice   a
+   2 │     2  Bob     b
+   3 │     3  Claire  c
+   4 │     4  Dylan   d
+
+julia> df[(df.x .> 2) .& (df.a .== 'c'), : ]
+1×3 DataFrame
+ Row │ x      y       a    
+     │ Int64  String  Char 
+─────┼─────────────────────
+   1 │     3  Claire  c
+```
+
+An alternative formulation, which notably saves on the need to use
```

Comment:
Now that we provide `subset` with a syntax closer to `select`, I'm not sure we should mention `filter` at all in Getting Started. Or maybe just mention it at the end for reference. What do you think @bkamins?

Reply:
We could do this - then it should be after `select` section as commented above. So - should I make this change?